src/main/kotlin/com/joetr/modulemaker/ModuleMakerDialogWrapper.kt

```diff
@@ -587,6 +587,7 @@ class ModuleMakerDialogWrapper(
         val currentlySelectedFile = getCurrentlySelectedFile()
         if (settingsGradleFile != null) {
             fileWriter.createModule(
+                rootPathString = selectedSrcValue.split(File.separator).drop(1).joinToString(File.separator),
```

Comment:
can you add a comment with this is doing with expected input?

something like

// at this point, selectedSrcValue has a value of something like `/Root/Module/Module2/` - we want to remove the root of the project to use as the file path in settings.gradle